core/shared/src/main/scala-2.13+/zio/ChunkBuilder.scala

```diff
@@ -363,11 +336,59 @@ object ChunkBuilder {
         case that: Short => self.arrayBuilder == that.arrayBuilder
         case _           => false
       }
+    def isEmpty: SBoolean =
+      arrayBuilder.length == 0
     def result(): Chunk[SShort] =
       Chunk.fromArray(arrayBuilder.result())
     override def sizeHint(n: SInt): Unit =
       arrayBuilder.sizeHint(n)
     override def toString: String =
       "ChunkBuilder.Short"
   }
+
+  final class Obj[A] extends ChunkBuilder[A] {
```

Comment:
Why did you choose `Obj` as the name for this class? Why not `ChunkBuilderLive` for example? 🤔

Reply:
Mostly because it's meant for objects (i.e., non-primitives). But yeah `Obj` doesn't sound too great. We could go with a similar name as `ArrayBuilder` and use `OfRef`

Reply:
Oh ok I understand now. Thanks! I let you choose what you prefer :)